index.bs

```diff
@@ -3428,58 +3477,72 @@ To <dfn>get the realm info</dfn> given |environment settings|:
 
 1. Let |realm id| be the [=realm id=] for |realm|.
 
+1. Let |origin| be the [=serialization of an origin=] given |environment settings|'s |origin|.
+
+1. Let |global object| be the [=Realm/global object=] specified by |environment settings|
+
 1. Run the steps under the first matching condition:
 
   <dl>
-    <dt>The [=Realm/global object=] specified by |environment settings| is a [=Window=] object
+    <dt>|global object| is a [=Window=] object
     <dd>
-     1. Let |type| be "<code>window</code>".
-
-    <dt>The [=Realm/global object=] specified by |environment settings| is a {{DedicatedWorkerGlobalScope}} object
+      1. Let |document| be |environment settings|' [=responsible document=].
+      1. Let |browsing context| be |document|'s [=browsing context=].
+      1. Let |context id| be the [=browsing context id=] for |browsing context|.
+      1. Let |realm info| be a map matching the <code>WindowRealmInfo</code> production,
+        with the <code>realm</code> field set to |realm id|, the <code>type</code>
+        field set to "<code>window</code>", the <code>origin</code> field set to |origin|,
```

Comment:
In other cases we haven't re-specified the value of fully constrained fields (i.e. ones whose value is a single literal) when constructing the object.